lte/gateway/python/integ_tests/s1aptests/s1ap_utils.py

```diff
@@ -1086,27 +1086,38 @@ def restart_sctpd(self, wait_time=30):
                 "/usr/local/bin/config_stateless_agw.py sctpd_pre';"
                 "docker start sctpd mobilityd pipelined sessiond oai_mme",
             )
-        self.wait_for_restart_to_finish(wait_time)
 
-    @staticmethod
-    def wait_for_restart_to_finish(wait_time):
-        for j in range(wait_time):
-            print(
-                f"Waiting for {wait_time - j} seconds for restart to complete",
-            )
-            time.sleep(1)
+    def wait_for_restart_to_finish(self, wait_time):
+        """wait for started services to become active or until timeout
+
+        Args:
+            services: (list[str]) list of services to check
```

Comment:
Just a nitpick: the doc-string mentions another argument `services`, which is not given (the service checks were moved to `check_if_magma_services_are_active`)